modules/core/src/main/java/org/apache/ignite/internal/processors/odbc/jdbc/JdbcConnectionContext.java

```diff
@@ -266,8 +266,10 @@ public JdbcConnectionContext(GridKernalContext ctx, GridNioSession ses, GridSpin
             }
         }
 
+        boolean serverBulkLoadEnabled = features.contains(JdbcThinFeature.SERVER_BULK_LOAD);
```

Comment:
it wrong place. You have JdbcConnectionContect inside JdbcRequestHandler and can use it like a 'connCtx.protocolContext().isFeatureSupported(.....)' instead of pass additional parameter to handler